src/attack.c

```diff
@@ -1421,8 +1421,15 @@ attack_network (struct scan_globals *globals, kb_t *network_kb)
   plugins_scheduler_free (sched);
 
   gettimeofday (&now, NULL);
-  g_message ("Total time to scan all hosts : %ld seconds",
-             now.tv_sec - then.tv_sec);
+  if (alive_hosts_list)
```

Comment:
If we use the new alive detection and 0 hosts are alive in the target range we won't print the new info because `alive_hosts_list` is NULL. 
I would just use `if (test_alive_hosts_only)`.

Reply:
I adopted your proposal.